### PR TITLE
feat(provider/dcos): Add the DC/OS load balancer wizard

### PR DESCRIPTION
**Pull request by willgorman**

### CODE REVIEW

File: app/scripts/modules/dcos/loadBalancer/configure/configure.dcos.module.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.configure.dcos', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/configure/configure.dcos.module.js]: `spinnaker.dcos.loadBalancer.configure`

File: app/scripts/modules/dcos/loadBalancer/configure/wizard/ports.controller.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.configure.dcos.ports', [])
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/configure/wizard/ports.controller.js]: `spinnaker.dcos.loadBalancer.configure.ports`

File: app/scripts/modules/dcos/loadBalancer/configure/wizard/resources.controller.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.configure.dcos.resources', [])
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/configure/wizard/resources.controller.js]: package name

File: app/scripts/modules/dcos/loadBalancer/configure/wizard/upsert.controller.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const angular = require('angular');
+
+import { ACCOUNT_SERVICE, LOAD_BALANCER_READ_SERVICE, LOAD_BALANCER_WRITE_SERVICE, V2_MODAL_WIZARD_SERVICE, TASK_MONITOR_BUILDER, SEARCH_SERVICE } from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.loadBalancer.dcos.create.controller', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/configure/wizard/upsert.controller.js]: package

File: app/scripts/modules/dcos/loadBalancer/details/details.controller.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+import { ACCOUNT_SERVICE, CONFIRMATION_MODAL_SERVICE, LOAD_BALANCER_WRITE_SERVICE, ServerGroupTemplates } from '@spinnaker/core';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.dcos.details.controller', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/details/details.controller.js]: package

File: app/scripts/modules/dcos/loadBalancer/details/details.controller.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+    $scope.state = {
+      loading: true
+    };
+
+    function extractLoadBalancer() {
+      $scope.loadBalancer = application.loadBalancers.data.filter(function (test) {
+        return test.name === loadBalancer.name &&
+          test.account === loadBalancer.accountId;
+      })[0];
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/details/details.controller.js]: You can replace the `.filter` above with `.find`, then omit the `[0]` here.

File: app/scripts/modules/dcos/loadBalancer/details/details.dcos.module.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.details.dcos', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/dcos/loadBalancer/details/details.dcos.module.js]: package

**Comment by willgorman** [comment]
Applied requested changes and rebased on master.

**Pull request merged**